mhkit/tidal/io/noaa.py

```diff
@@ -205,8 +208,7 @@ def request_noaa_data(
         handle_caching(
             hash_params,
             cache_dir,
-            data=data,
-            metadata=metadata,
+            cache_content={"data": data, "metadata": metadata, "write_json": None},
             clear_cache_file=clear_cache,
         )
```

Comment:
handle_caching reduced to only 5 inputs
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-args